src/store.ts

```diff
@@ -3,23 +3,23 @@ import { createLogger } from 'redux-logger';
 import thunk from 'redux-thunk';
 import { posts, PostsState } from './modules/posts/reducer';
 import { albums, AlbumsState } from './modules/albums/reducer';
-import { GithubRepositoryState, gitHubRepositories } from './modules/github/reducer';
-import {gitHubDetailRepositories, GithubDetailRepositoryState} from "./modules/githubDetailRepository/reducer";
+import { GithubRepositoriesState, gitHubRepositories } from './modules/github/reducer';
+import { githubRepository, GithubDetailRepositoryState} from "./modules/githubDetailRepository/reducer";
 
 const loggerMiddleware = createLogger();
```

Comment:
Better use redux devtools browser extension (it comes with a Redux midddleware). Then browser console won't be flooded with logs.